src/pages/TodayPage/Components/CreateReminder.tsx

```diff
@@ -69,8 +69,8 @@ function CreateReminder() {
       });
 
     setToDoData({
-      title: "",
-      description: "",
+      title: titleRef.current?.value,
+      description: descriptionRef.current?.value,
```

Comment:
We are resetting fields after successfully submission, that's why those fields should be empty
```suggestion
      title: "",
      description: "",
```